crates/sui-indexer/src/schema/pg.rs

```diff
@@ -223,52 +223,83 @@ diesel::table! {
 }
 
 diesel::table! {
-    tx_calls (package, tx_sequence_number, cp_sequence_number) {
-        cp_sequence_number -> Int8,
+    tx_calls_pkg (package, tx_sequence_number) {
+        package -> Bytea,
         tx_sequence_number -> Int8,
+        sender -> Bytea,
+    }
+}
+
+diesel::table! {
+    tx_calls_mod (package, module, tx_sequence_number) {
         package -> Bytea,
         module -> Text,
-        func -> Text,
+        tx_sequence_number -> Int8,
+        sender -> Bytea,
     }
 }
 
 diesel::table! {
-    tx_changed_objects (object_id, tx_sequence_number, cp_sequence_number) {
-        cp_sequence_number -> Int8,
+    tx_calls_fun (package, module, func, tx_sequence_number) {
```

Comment:
(Optional) For the sake of backwards compatibility, you may want to keep this one called `tx_calls` -- benefit is probably minimal as you may have to account for the lack of `cp_sequence_number` everywhere.